README.md

```diff
@@ -19,20 +20,27 @@ TBD
 
 4. demo usage of the API: many nano examples that use these APIs to solve real (perhaps trivial) problems
 
-
 ## Quick start
 
 Install `docker`.
 
 Build the image, see details in the `dockerfile`.
 
-`docker build -t gee:latest .`
+```bash
+docker build -t gee:latest .
+```
 
 Run the container in Mac/Linux host
-`docker run --name "gee" -it -p "127.0.0.1:8888:8888" -v ./workspace:/app gee`
+
+```bash
+docker run --name "gee" -it -p "127.0.0.1:8888:8888" -v ./workspace:/app gee
```

Comment:
@Yichuans  Can we please add some more info on what those parameters are?
I was trying to run it from the main app folder, but even using `-v ./` or `-v .` was returning an error. While running the correct command from outside the main folder and pointing to that from `-v` has worked fine.

Moreover, there might be some instances of "gee" running, so there are some commands we could use to manage that:

- `docker stop gee` where gee is the name of the container
- `docker rm gee` which removes the container once it has been stopped
- `docker start gee` which runs an existing container named gee which has been previously stopped